armory/included/reports/HostSummary.py

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+from armory.included.ReportTemplate import ReportTemplate
+from armory.database.repositories import IPRepository
+import pdb
+import json
+from fuzzywuzzy import process
+import datetime
+import glob
+import xmltodict
+import os
+from jinja2 import Template
+from base64 import b64encode
+import shutil
+
+sev_map = {0: 'Informational', 1: 'Low', 2:'Medium', 3:'High', 4:'Critical'}
+
+class Report(ReportTemplate):
+    '''
+    This report displays the various HTTP header findings we have
+    '''
+
+    name = "HeaderFindings"
```

Comment:
May want to change this to be `HostSummary`, don't remember if the `Report.name` has any uses.